src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Layout, Row, Col, Input } from 'antd';
+import { MenuOutlined, LoginOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+
+const { Header: AntHeader } = Layout;
+const { Search: AntSearch } = Input;
+
+const StyledHeader = styled(AntHeader)`
+  padding: 0 2rem 0 0;
+`;
+
+const StyledMenuOutlined = styled(MenuOutlined)`
+  font-size: 1rem;
+  color: white;
+`;
+
+const StyledSearch = styled(AntSearch)`
+  max-width: 800px;
+  margin: 0px 1rem;
+`;
+
+const Center = styled(Col)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
Good use of the styled-components, if the styles keep growing, consider move them to their own file, something like  `Header.styles.jsx`